### PR TITLE
Update Document Header-Part-1

### PR DESCRIPTION
**Pull request by tahmeedshah**
### General description
<=============== PLEASE DON'T MERGE!=================>
Updated the document-loader directive to incorporate all the buttons displayed in the document header. Additionally, brought the schema title from common-format view pages and the close button from the result default page into the record-loader directive. Furthermore, repositioned the buttons in the record-loader directive.

### Designs
_Link to the related design prototypes (if applicable)_

### Testing instructions
Test for CNA records only
* Apart from the added feature / bug fix, check overall performance, styling...

## Checklist before merging
* [ ]  Branch name / PR includes the related Jira ticket Id.
* [ ]  Tests to check core implementation / bug fix added.
* [ ]  All checks in Continuous Integration workflow pass.
* [ ]  Feature functionally tested by reviewer(s).
* [ ]  Code reviewed by reviewer(s).
* [ ]  Documentation updated (README, CHANGELOG...) (if required)

### CODE REVIEW

File: app/views/forms/view/record-loader.directive.html [hunk 1]
```diff
@@ -108,62 +108,86 @@ <h3>
...
-        </div>
-        <div id="sendRecords" ng-if="shareVueComponent && internalDocumentInfo" class="share-single-document full-text-display pe-2 float-end compare-buttons">
-            <span class="color-white">|&nbsp;</span>
-            <share-record ng-vue="shareVueComponent" :get-query="getQuery" :token-reader="tokenReader" :user-status="userStatus"></share-record>
-        </div>
-        <div class="float-end pe-2 position-relative text-white" style="top:7px;" ng-show="!loading && !error && internalDocumentInfo">
-            <div record-options></div>
-        </div>
+        <!-- Document Header -->
+        <!-- use schema.type for condition  -->
```
Comment (blaisefonseca) [inline app/views/forms/view/record-loader.directive.html:112]: not so useful comments

File: app/views/forms/view/record-loader.directive.html [hunk 2]
```diff
@@ -108,62 +108,86 @@ <h3>
...
+                        </div>
+                    </div> 
+                    <!-- Compare button  -->
+                    <div ng-if="showDifferenceButton||showComparison" class="full-text-display float-end compare-buttons pe-2 position-relative text-white">
+                        <button type="button" ng-if="showDifferenceButton" ng-disabled="isComparing" class="btn btn-success btn-sm"
+                            ng-click="showDifference()">
+                            <i class="fa" ng-class="{'fa-check-square-o':showDifferenceOn, 'fa-square-o': !showDifferenceOn}"></i> {{::('recordLoaderT.showDifference'|$translate)}}
+                        </button>
+                        <!-- for comparing with older versions -->
+                        <!-- ng-if="showDifferenceButton"  style="display: none;"-->
```
Comment (blaisefonseca) [inline app/views/forms/view/record-loader.directive.html:150]: delete comment which are not required

File: app/views/forms/view/record-loader.directive.js [hunk 3]
```diff
@@ -40,6 +40,7 @@ const sleep = (ms)=>new Promise((resolve)=>setTimeout(resolve, ms));
 				locale: "=?",
 				hide: "@",
 				showDetails: "=",
+				isClose: "<", // to hide the close button on single record view
```
Comment (blaisefonseca) [inline app/views/forms/view/record-loader.directive.js:43]: is this still required?

File: app/views/forms/view/record-loader.directive.js [hunk 4]
```diff
@@ -74,6 +75,10 @@ const sleep = (ms)=>new Promise((resolve)=>setTimeout(resolve, ms));
 								$scope.revisionNo = 'draft'
 						}
 					});
+					//close record in result default page
+					$scope.closeDoc = function(){
+						$scope.$emit('recordClosedEvent', false);
```
Comment (blaisefonseca) [inline app/views/forms/view/record-loader.directive.js:80]: please look at the the event format in other places

File: app/views/forms/view/record-loader.directive.js (hunk 3 above)
Comment (tahmeedshah) [reply app/views/forms/view/record-loader.directive.js:43]: Yes, it is necessary to hide the close button when a single record is opened.
For example: https://bch.cbddev.xyz/en/database/CNA/BCH-CNA-VA-266688-1

**Review by sbilodeau — APPROVED** [review]

**Review by blaisefonseca — APPROVED** [review]

**Pull request merged**